src/main/java/org/cyclops/integratedtunnels/capability/network/TunnelNetworkCapabilityConstructors.java

```diff
@@ -26,20 +28,22 @@
     @SubscribeEvent
     public void onNetworkLoad(AttachCapabilitiesEventNetwork event) {
         ItemNetwork itemNetwork = new ItemNetwork();
+        IItemChannel itemChannel = itemNetwork.getChannel(IChanneledNetwork.DEFAULT_CHANNEL);
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "itemNetwork"),
                 new DefaultCapabilityProvider<IItemNetwork>(() -> ItemNetworkConfig.CAPABILITY, itemNetwork));
         event.addCapability(new ResourceLocation(Reference.MOD_ID, "itemStorageNetwork"),
-                new DefaultCapabilityProvider<IItemHandler>(() -> CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, itemNetwork));
+                new DefaultCapabilityProvider<IItemHandler>(() -> CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, itemChannel));
```

Comment:
As per https://github.com/CyclopsMC/IntegratedDynamics/pull/332/files#r155936160, do we even need these capabilities for anything?

Reply:
Answered in the referenced comment.